### PR TITLE
Update README.md (typos, VS Codespaces)

### PR DESCRIPTION
**Pull request by kimadeline**
- Fix some typos
- VS Codespaces instead of VS Online

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 
-The scenario is if you are developing in a remote environment--like WSL, a container, ssh, or [VS Online](https://visualstudio.microsoft.com/services/visual-studio-online/)--and are port-forwarding a server to develop (and debug) in a browser locally. For remote development, VS Code runs two sets of extensions: one on the remote machine, and one on your local computer. `js-debug` is a "workspace" extension that runs on the remote machine, but we need to launch and talk to Chrome locally.
+The scenario is if you are developing in a remote environment—like WSL, a container, ssh, or [VS Codespaces](https://visualstudio.microsoft.com/services/visual-studio-codespaces/)—and are port-forwarding a server to develop (and debug) in a browser locally. For remote development, VS Code runs two sets of extensions: one on the remote machine, and one on your local computer. `js-debug` is a "workspace" extension that runs on the remote machine, but we need to launch and talk to Chrome locally.
```
Comment (connor4312) [inline README.md:5]: Personally I would leave the styling of em-dashes up to the renderer, but this is super nitpicky and Github doesn't em-dash correctly 😄

**Comment by connor4312** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (kimadeline) [reply README.md:5]: Would you prefer the em-dashes change to be reverted? Can do too 🙂

Comment (connor4312) [reply README.md:5]: It's good as-is :)

**Pull request merged**